uber/config.py

```diff
@@ -396,7 +392,10 @@ def _unrepr(d):
 c.PRICE_LIMITS = {}
 for _opt, _val in c.BADGE_PRICES['attendee'].items():
     try:
-        date = c.EVENT_TIMEZONE.localize(datetime.strptime(_opt, '%Y-%m-%d'))
+        if ' ' in _opt:
```

Comment:
This allows us to specify an hour and minute for price bumps, which is required if we're going to set price bumps retroactively to match when our prices were raised by the badge cap.